core/src/main/kotlin/plugins/OpenApi.kt

```diff
@@ -79,7 +79,8 @@ fun Application.configureOpenApi() {
         }
 
         server {
-            url = "http://localhost:8080"
+            val port = System.getenv("ORT_SERVER_CORE_PORT") ?: "8080"
+            url = "http://localhost:$port"
```

Comment:
If you change the port here you also have to adapt the port mapping in docker-compose.yml.

Also, I think the actual issue is that the port that the generated code uses cannot be set at runtime, if this cannot be fixed, this variable will never work without rebuilding the docker image for the UI, once we have it.

Having to set the port (and URL) at compile time makes it impossible to publish reusable binaries.

Reply:
This could help: https://github.com/7nohe/openapi-react-query-codegen/issues/94#issuecomment-2079344928

Reply:
Let me just drop this hunk instead. My main point anyway was to make `pnpm codegen` use the correct port.